CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#6829](https://github.com/CocoaPods/CocoaPods/pull/6829)
 
+* Provide a better error message when references are missing for non-source files
```

Comment:
@davidair this is a nit but can you add two empty spaces at the end of this line? Like the other CHANGELOG entries?